modules/rosa-create-objects.adoc

```diff
@@ -861,6 +861,73 @@ Add a machine pool with tags to a cluster.
 $ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --tags='foo bar,bar baz' --name=mp-1
 ----
 
+[id="rosa-create-network_{context}"]
+== create network
+
+Create a network that creates any necessary AWS resources through AWS CloudFormation templates. This helper command is intended to help create and configure a VPC for use with ROSA with HCP.
+
+[NOTE]
+====
+For custom or advanced configuration, it is highly recommended to use the AWS CLI directly using the `aws cloudformation` command.
+====
+
+[IMPORTANT]
+====
+Running this command creates resources within your AWS account.
+====
+
+.Syntax
+[source,terminal]
+----
+$ rosa create network [flags]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|<template-name>
+|Allows you to use a custom template. Templates must be in the template folder, structured as _templates/<template-name>/cloudformation.yaml_. If no template name is provided, command uses the default template.
+
+|===
+
+.Default template YAML
+[source,yaml]
+----
+include::https://raw.githubusercontent.com/openshift/rosa/refs/heads/master/cmd/create/network/templates/rosa-quickstart-default-vpc/cloudformation.yaml[]
+----
+
+.Flags
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--template-dir
+|Allows you to specify the path to the template directory. Overrides the OCM_TEMPLATE_DIR environment variable. Required if not running the command inside the template directory.
```

Comment:
@den-rgb From the last night as we discussed, I guess now --template-dir wouldn't be required if not running inside template directory, Could you please confirm once here the changes ? Thanks

Reply:
THis will be necessary until I am certain I can find a way to reference it as a default through binary builds